gruntfile.js

```diff
@@ -23,8 +25,8 @@ module.exports = function (grunt) {
     'if:linuxAmd64BinaryNotExist',
     'html2js',
     'useminPrepare:dev',
-    'recess:build',
     'concat',
+    'postcss:build',
```

Comment:
Minification / vendor prefixes should not be required for the `build` task. It's used for local development only.